### DISCUSSION TITLE
BC2D7B60-20F6-4C22-9ED6-D6110B376063

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#96] [General]
# K2-384

K2-384 is a red dwarf located about 269.47 lightyears form the solar system. It hosts a compact system of five planets: an Earth-sized world (b= 1.076 Re), three super-Earths (c= 1.191 Re | d= 1.392 Re | e= 1.345 Re) and a mini-Neptune (f= 2.222 Re). None of their masses are known.

https://celestia.mobi/addon?item=BC2D7B60-20F6-4C22-9ED6-D6110B376063&lang=en

**Comment by exocritic5457** [comment]
There's a place in my heart for obscure multiplanetary systems.
